transaction/transaction.go

```diff
@@ -13,9 +13,10 @@ import (
 )
 
 const (
-	LOGSIZE     int = 4 * 1024 * 1024
-	LBUFFERSIZE     = 512 * 1024
-	BUFFERSIZE      = 4 * 1024
+	LBUFFERSIZE = 512 * 1024 // TODO: This could be a problem in the future
+	MAGIC       = 131071
```

Comment:
As Rajesh mentioned in another review for go-pmem, let's use a bigger number as the magic constant so as to maximize the number of actual bits we use to store this constant. You can run uuidgen or similar random number generator to generate a big random number.